tests/test_mem_eff_attention.py

```diff
@@ -724,3 +724,43 @@ def test_memory_efficient_attention_full_block_masked(
     assert_allclose(grad_q, query.grad, "grad_q", atol=atol)
     assert_allclose(grad_k, key.grad, "grad_k", atol=atol)
     assert_allclose(grad_v, value.grad, "grad_v", atol=atol)
+
+
+@pytest.mark.parametrize("dim", [0, 1, 2, 3, 4])
+def test_unbind(dim):
+    x = torch.ones([10, 20, 4, 10, 3], requires_grad=True)
+    x2 = torch.ones([10, 20, 4, 10, 3], requires_grad=True)
+
+    # FW
+    tensors = xformers.ops.unbind(x, dim)
+    tensors2 = torch.unbind(x2, dim)
+    assert len(tensors) == len(tensors2)
+    for t1, t2 in zip(tensors, tensors2):
+        assert torch.allclose(t1, t2)
+
+    # BW
+    grads = torch.unbind(torch.randn(x.shape), dim)
+    loss1 = sum(g * t for (g, t) in zip(grads, tensors))
+    loss2 = sum(g * t for (g, t) in zip(grads, tensors2))
+    assert torch.allclose(loss1, loss2)
+    g = torch.ones_like(loss1)
```

Comment:
Instead of making `ones_like`, can you make instead `torch.randn_like` ? This will make the gradients not be all ones, which will help test this function